### PR TITLE
caldav binding 26/11

### PR DESCRIPTION
**Pull request by querdenker2k**

### CODE REVIEW

File: bundles/binding/org.openhab.binding.caldav-personal/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Bundle-Name: openHAB CalDav Binding
+Bundle-SymbolicName: org.openhab.binding.caldav-personal;singleton:=true
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.8.0.qualifier
+Bundle-Activator: org.openhab.binding.caldav_personal.internal.CalDavActivator
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the CalDav-Personal binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: org.apache.commons.lang,
+ org.apache.commons.lang.math;version="2.6.0",
```
Comment (hakan42) [inline bundles/binding/org.openhab.binding.caldav-personal/META-INF/MANIFEST.MF]: Please remove specific version if not necessary, else document it.

**Comment by querdenker2k** [comment]
Is it merged? nice. thanks a lot.

finally i just have to say once again: git is not my friend.

**Comment by hakan42** [comment]
No problem, git hates me too. I am fixing a few little things, then I'll write another note here.

**Comment by hakan42** [comment]
@querdenker2k , thank you very much for your persistent efforts to get this binding created. Especially for creating five different pull requests as git seems to hate us all :smile: 

Now all we need is a wiki page and an announcement on the forums tomorrow when the jenkins build is available.

**Comment by querdenker2k** [comment]
Wiki is already available: https://github.com/openhab/openhab/wiki/CalDAV

**Comment by teichsta** [comment]
@querdenker2k @hakan42 thanks for driving this forward!

Please also add the new bundles to:

* bundles/binding/pom.xml
* bundles/io/pom.xml
* bundles/persistence/pom.xml
* distribution/pom.xml

otherwise the bundles will not be included in the build and hence not be part of the distribution

**Comment by hakan42** [comment]
Ah, thank you. I added to the bundles yesterday night, but forgot distribution. Fresh PR coming in fayf minüts :smile:

**Comment by hakan42** [comment]
o.k., the distribution was added too, tonights build should contain all the jars.

**Comment by teichsta** [comment]
ah, wasn't aware of these PR's thanks!

**Comment by querdenker2k** [comment]
Oh sorry did i really forgot them all...
Thanks for adding

**Comment by querdenker2k** [comment]
Binding is tested with openHAB 2 and working (except persistence binding wasn't tested)

**Comment by hakan42** [comment]
Thank you, I will create a PR to include the confirmed parts tomorrow.

If the persistence part is not yet tested, I would leave this for another PR later on, after we collect some experience with the confirmed parts.

Let's wait for the generic JDBC binding first, and create PRs one after another :smile:

**Pull request merged**